Clinical-Genomics/cgweb
# Orderform 1306:7, small o in other

How to test:
0. install on stage (only cgweb)
1. go to OP-stage
2. upload xlsx for orderform with "other (specify in comment field)" selected for your sample DNA extraction method

Expected outcome:
Sample shown in OP but collapsed (=successfully loaded)
